htdocs/takepos/admin/setup.php

```diff
@@ -95,6 +95,13 @@
 	}
 
 	if (!$error) {
+		// add terminal permissions
+		require_once DOL_DOCUMENT_ROOT . '/core/modules/modTakePos.class.php';
```

Comment:
Using 1 permission to assign a different user on each terminal is not always the desired behaviour. Small shops would prefer to have one global permission as currently. So can you introduce first a hidden option to switch in mode 1 permission per terminal. 
When stable we will be able later to offer the option on takepos setup...

Reply:
ok @eldy i introduce TAKEPOS_USE_TERMINAL_PERMISSIONS